docs/admin/cluster-management.md

```diff
@@ -80,6 +72,37 @@ If you want more control over the upgrading process, you may use the following w
     be created automatically when you create a new VM instance (if you're using a cloud provider that supports
     node discovery; currently this is only Google Compute Engine, not including CoreOS on Google Compute Engine using kube-register). See [Node](node.md).
 
+## Advanced Topics
+
+### Upgrading to a different API version
+When a new API is released, you may need to upgrade a cluster to support a new API version (e.g. switching from 'v1' to 'v2' when 'v2' is launched)
+
+This is an infrequent event, but it requires careful management. There is a sequence of steps to upgrade to a new API version.
+
+1. Turn on the new api version
+2. Upgrade the cluster's storage to use the new version.
+3. Upgrade all config files. Identify users of the old api version endpoints.
+4. Update existing objects in the storage to new version by running cluster/update-storage-objects.sh
+3. Turn off the old version.
+
+### Turn on or off an API version for your cluster
+
+Specific API versions can be turned on or off by passing --runtime-config=api/<version> flag while bringing up the server. For example: to turn off v1 API, pass --runtime-config=api/v1=false.
```

Comment:
I'd add code quotes to both instances of --runtime-config=api/<version> so they look like `--runtime-config=api/<version>`